scripts/opentitan/romtool.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2025 Rivos, Inc.
+# SPDX-License-Identifier: Apache2
+
+"""QEMU OT tool to generate a scrambled ROM image.
+
+   :author: Emmanuel Blot <eblot@rivosinc.com>
+"""
+
+# pylint: disable=invalid-name
+# pylint: enable=invalid-name
+
+from argparse import ArgumentParser, FileType
+from os.path import dirname, join as joinpath, normpath
+from traceback import format_exception
+import sys
+
+QEMU_PYPATH = joinpath(dirname(dirname(dirname(normpath(__file__)))),
+                       'python', 'qemu')
+sys.path.append(QEMU_PYPATH)
+
+# ruff: noqa: E402
+from ot.rom.image import ROMImage
+from ot.util.arg import ArgError
+from ot.util.log import configure_loggers
+from ot.util.misc import HexInt
+
+
+def main():
+    """Main routine"""
+    debug = True
+    desc = sys.modules[__name__].__doc__.split('.', 1)[0].strip()
+    argparser = ArgumentParser(description=f'{desc}.')
+    out_formats = list(ROMImage.save_formats)
+    # make hex format, if supported, the first (default) one
+    out_formats.sort(key=lambda x: x if x != 'HEX' else '')
+    try:
+
+        files = argparser.add_argument_group(title='Files')
+        files.add_argument('-c', '--config', type=FileType('rt'),
+                           metavar='CFG', required=True,
+                           help='input QEMU OT config file')
+        files.add_argument('rom', nargs=1, type=FileType('rb'),
+                           help='input ROM image file')
+        files.add_argument('-o', '--output',
+                           help='output ROM image file')
+
+        params = argparser.add_argument_group(title='Parameters')
+        params.add_argument('-i', '--rom-id', type=int,
+                            help='ROM image identifier')
+        params.add_argument('-z', '--rom-size', metavar='SIZE',
+                            type=HexInt.xparse,
+                            help='ROM image size in bytes, may be repeated')
```

Comment:
Bad copy/paste on doc comment?
```suggestion
                            help='ROM image size in bytes')
```

Reply:
It seems so :-(